CONTRIBUTING.md

```diff
@@ -12,7 +12,68 @@ Please consult the [README](./README.md) file at the root of this repository.
 
 ## How to Contribute
 
-Please follow the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website.
+Please read the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website. It contains some general instructions that should be followed when contributing to any of the Creative Commons open-source repositories.
+
+### Bugs
+
+If you find a bug, please open an issue in this repository describing the bug. The bug description should contain:
+
+* Step-by-step to reproduce
+* What the expected behaviour is
+* Which browsers, including their versions, and operating system were used to reproduce the bug
+
+Example of a good bug description:
+
+```
+License filters don't work as expected
+
+How to reproduce:
+
+- Go to home page
+- Search for any term (eg: nature)
+- In the search results page, click on Filter and select CC-BY-SA on the "All licenses" select
+- Click on the third image (<link to image>)
+- The image isn't licensed as CC-BY-SA but as CC-BY-NC instead
+
+Expected behaviour
+
+- Only images licensed as CC-BY-SA should be displayed in the search results page
+
+Browser used:
+
+Safari 12.0.3, MacOS 10.14.3
+Firefox 66.0.2, MacOS 10.14.3
+```
+
+After that, don't forget to tag the issue with the "Bug" label.
+
+### Proposing changes or new features
+
+If you have an idea of a new feature or change to how the CC Search frontend works, please file an issue so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
+
+### Pull requests
+
+Before you start writing code, make sure there is an issue open. Pull requests without a link to an existing issue won't be merged.
+
+If you want to get started contributing code to this project but don't know exactly what to work on, we compiled a good list of issues labeled as [Good first issues](https://github.com/creativecommons/cccatalog-frontend/labels/good%20first%20issue) which are small in scope and not so complex to solve. There is also issues labeld as [Help wanted](https://github.com/creativecommons/cccatalog-frontend/labels/help%20wanted) which can be a bit more complex but are good examples of thigns we are currently accepting help from the community.
+
+Once you open your Pull Request, make sure that its description contains the following information:
```

Comment:
I think this should go in the PR template.

Reply:
In the #175 issue description it says "Remove custom issue/pull request templates". Are we keeping them?

Reply:
No, we should remove them here, they will be automatically pulled in from the ones in https://github.com/creativecommons/.github.

Reply:
See https://wikijs.creativecommons.org/tech/github-repo-guidelines (only accessible to CC staff) for more details.